config/routes.rb

```diff
@@ -1,60 +1,16 @@
 Rails.application.routes.draw do
 
+  root 'companies#index'
+
   resources :companies do
-    resources :jobs
+    resources :contacts, only: [:create]
+    resources :jobs do
+      resources :comments, only: [:create]
```

Comment:
avoid nesting 3 deep...only 2...could pull `comments` out and nest comments in jobs on its own